timers.js

```diff
@@ -1,4 +1,38 @@
 
+var reset = document.getElementById('reset');
+var start = document.getElementById('start');
+var pause = document.getElementById('pause');
+var timer = document.getElementById('timer');
 
+var seconds = 0;
+var timerId;
 
+var resetListener = reset.addEventListener('click', resetButton());
```

Comment:
remove parens from `resetButton`. Once the click happens, then invoke that function.